src/System.Binary.Base64/System.Binary.Base64.csproj

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  <PropertyGroup>
+    <ProjectGuid>b9ecbaf7-524d-4134-ae37-eff7802b221f</ProjectGuid>
```

Comment:
You will need to pull in a newer CLI in order to use the new SDK syntax. I think CoreFxLab would be a great candidate for dogfooding that build portion of the CLI, that is a little trickier to to in CoreFx right now.